lldb/source/Plugins/Process/AIX/NativeProcessAIX.cpp

```diff
@@ -0,0 +1,265 @@
+//===-- NativeProcessAIX.cpp --------------------------------------------===//
```

Comment:
Pad with more `-` so that the line is the same length as the next `//===---` line.